src/Nethermind/Nethermind.Merge.Plugin/EngineRpcModule.Paris.cs

```diff
@@ -70,29 +72,33 @@ private async Task<ResultWrapper<PayloadStatusV1>> NewPayload(ExecutionPayload e
             return ResultWrapper<PayloadStatusV1>.Fail(error, ErrorCodes.InvalidParams);
         }
 
-        if (await _locker.WaitAsync(_timeout))
+        using (_gcKeeper.TryStartNoGCRegion())
```

Comment:
NP: Use the `using` syntax without the bracket. Keeps code to left, and make things easier to review.

Reply:
Ok I need to keep it under lock though